.github/workflows/build.yml

```diff
@@ -22,16 +22,20 @@ jobs:
     container:
       image: quay.io/ovirt/buildcontainer:${{ matrix.container-name }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - name: Install testing dependencies
         run: |
+          dnf install -y --setopt=tsflags=nodocs git
```

Comment:
why?
podman run --rm -it quay.io/ovirt/buildcontainer:el8stream rpm -q git
git-2.31.1-2.el8.x86_64

Reply:
ok, dropped